low-level/sensors/impl/centos_7/service_monitor.py

```diff
@@ -80,6 +81,7 @@ class FailedState:
 
     @staticmethod
     def enter(service):
+        service.state_enter_timestamp = time.time()
```

Comment:
I am not able to understand why enter() is implemented twice? although it has same functionality

Reply:
for each service state we have defined  class(ActiveState,FailedState, InactiveState etc) and for each class we have defined enter() method.
So for eg,  If service state changes to "active",  enter() method from ActiveState class get called and take necessary action.